alicat/serial.py

```diff
@@ -132,21 +132,175 @@ def set_gas(self, gas, retries=2):
         """Set the gas type.
 
         Args:
-            gas: The gas type, as a string. Supported gas types are:
+            gas: The gas type, as a string or integer. Supported gas types by string are:
                 'Air', 'Ar', 'CH4', 'CO', 'CO2', 'C2H6', 'H2', 'He', 'N2',
                 'N2O', 'Ne', 'O2', 'C3H8', 'n-C4H10', 'C2H2', 'C2H4',
                 'i-C2H10', 'Kr', 'Xe', 'SF6', 'C-25', 'C-10', 'C-8', 'C-2',
                 'C-75', 'A-75', 'A-25', 'A1025', 'Star29', 'P-5'
+
+                For the full gas table, please see page 52 of the controller manual here:
+                https://documents.alicat.com/manuals/DOC-MANUAL-MC.pdf
+
+                Gas mixes may only be called by their mix number.
         """
```

Comment:
Could you do me a favor with code style?

```
pip install pycodestyle pydocstyle
pycodestyle path/to/alicat/*.py
pydocstyle path/to/alicat/*.py
```

They'll yell at you about a bunch of minutiae. Could you go through and fix these?

(There are also code editors with these built in so you don't need to use the command line. I use Atom, which is free and cross-platform.)